agent/engine/task_manager.go

```diff
@@ -130,8 +130,13 @@ func (mtask *managedTask) overseeTask() {
 			llog.Debug("Task not steady state or terminal; progressing it")
 
 			// TODO:
-			// 1) Add platform resources setup and failure handling
-			// 2) Add new task resources provisioned state ?
+			// 1) Add new task resources provisioned state ?
```

Comment:
I'd recommend create an issue for this if we plan to address this before the release.